node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,10 @@
+import { writeFile } from 'node:fs';
+import { Buffer } from 'node:buffer';
+
+const data = new Uint8Array(Buffer.from('Hello Node.js'));
+writeFile('./random.txt', data, (err) => {
+  if (err) throw err;
+  Math.random();
```

Comment:
I'm also not sure why this is here.

I think you want to assign a random number to `data`.